multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -238,6 +239,50 @@ def is_image_column(
         return False
 
 
+def is_document_pdf_column(
+    data: pd.Series,
+    col_name: str,
+    sample_m: Optional[int] = 500,
+) -> bool:
+    """
+    Identify if a column is a pdf document column.
+
+    Parameters
+    ----------
+    data
+        One column of a multimodal pd.DataFrame for training.
+    col_name
+        Name of column.
+    sample_m
+        Number of sample images used to check if images are documents images.
+
+    Returns
+    -------
+    Whether the column is a pdf document column.
+    """
+
+    if len(data) > sample_m:
+        # Sample to speed-up type inference
+        data = data.sample(n=sample_m, random_state=0)
+    is_all_pdf = []
+
+    for docs in data:
+        try:
+            if not isinstance(docs, list):
+                docs = [docs]
+            for per_doc in docs:
+                if per_doc.endswith(".pdf"):
+                    is_all_pdf.append(True)
+                else:
+                    is_all_pdf.append(False)
```

Comment:
Can we just return False in this case? The below logic requires all True.

Reply:
sounds good

Reply:
fixed in the new commit.